corehq/apps/style/templates/style/includes/keyboard_shortcuts.html

```diff
@@ -0,0 +1,227 @@
+                                <option value="option" {% if request.couch_user.keyboard_shortcuts.main_key == 'option' %} selected{% endif %}>option / alt</option>
+                                <option {% if request.couch_user.keyboard_shortcuts.main_key == 'ctrl' %} selected{% endif %}>ctrl</option>
+                                <option {% if request.couch_user.keyboard_shortcuts.main_key == 'command' %} selected{% endif %}>command</option>
+                                <option {% if request.couch_user.keyboard_shortcuts.main_key == 'shift' %} selected{% endif %}>shift</option>
+                            </select>
+                        </div>
+                    </div>
+                    <div class="alert alert-info">
+                        {% trans "For more information visit: " %}
+                        <a href="https://help.commcarehq.org/display/commcarepublic/CommCareHQ+Keyboard+Shortcuts">{% trans "CommCare HQ Keyboard Shortcuts Wiki" %}</a>
+                    </div>
+                    <hr />
+                    <ul class="unstyled" id="keyboard-shortcuts-lookup"></ul>
+                    <strong>{% trans "While navigating menus or the sidebar" %}:</strong>
+                    <ul class="unstyled">
+                        <li>{% trans "Move up" %}: <code>{{ request.couch_user.keyboard_shortcuts.main_key|default:"*shortcut key*" }}+k</code> {% trans "or" %} <code>{{ request.couch_user.keyboard_shortcuts.main_key|default:"*shortcut key*" }}+up</code></li>
+                        <li>{% trans "Move down" %}: <code>{{ request.couch_user.keyboard_shortcuts.main_key|default:"*shortcut key*" }}+j</code> {% trans "or" %} <code>{{ request.couch_user.keyboard_shortcuts.main_key|default:"*shortcut key*" }}+down</code></li>
+                        <li>{% trans "Do action (e.g. browsing to a page)" %}: <code>{{ request.couch_user.keyboard_shortcuts.main_key|default:"*shortcut key*" }}+enter</code> {% trans "or" %} <code>{{ request.couch_user.keyboard_shortcuts.main_key|default:"*shortcut key*" }}+spacebar</code></li>
+                    </ul>
+                </div>
+                <div class="modal-footer">
+                     <button class="btn btn-primary" type="submit">
+                         {% trans "Update" %}
+                     </button>
+                </div>
+            </form>
+
+        </div>
+    </div>
+</div>
+
+{% compress js %}
+<script src="{% new_static 'keymaster/keymaster.js' %}"></script>
+{% endcompress %}
+<script>
+    var KEY = key.noConflict();  // Make global key function capital to adhere to globals convention
+                                 // and to avoid conflicts in other places where "key" is used as variable name
+</script>
+
+{% if request.couch_user.keyboard_shortcuts.enabled %}
+    {% compress js %}
+    <script src="{% new_static 'hqwebapp/js/keyboard_navigator.js' %}"></script>
+    {% endcompress %}
+{% endif %}
+
+<script>
+var KS = {
```

Comment:
I think that's definitely a good move. I'm just copying what was in the b2 template so that the issue of no keyboard shortcuts can at least be addressed in the interim.